maintenance_account/tests/test_maintenance_account.py

```diff
@@ -92,7 +95,9 @@ def test_invoice_action_post_equipment_1(self):
         self.assertEqual(len(line_a.equipment_ids), 2)
         self.assertEqual(len(line_b.equipment_ids), 0)
         equipment = fields.first(equipments)
-        self.assertEqual(equipment.name, self.product_a.name)
+        name_list = line_a.name.split("\n")
+        self.assertEqual(equipment.name, name_list[0])
+        self.assertEqual(html2plaintext(equipment.note), name_list[1])
```

Comment:
Don't test code. Test fixed data:

```suggestion
        self.assertEqual(equipment.name, "Product A")
        self.assertEqual(equipment.note, "<p>Test description product A</p>")
```